proto/cosmos/bank/v1beta1/bank.proto

```diff
@@ -43,6 +43,21 @@ message Output {
       [(gogoproto.nullable) = false, (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins"];
 }
 
+// Supply represents a struct that passively keeps track of the total supply
+// amounts in the network.
+// This message is deprecated now that supply is indexed by denom.
+message Supply {
+  option deprecated = true;
```

Comment:
ref: https://github.com/cosmos/cosmos-sdk/pull/8517#discussion_r587339285

Reply:
Is it documented somewhere how we want to deal with changes in proto. As I understand it, we can make changes as it's still labellled beta. wdyt?

Reply:
Yes, https://github.com/cosmos/cosmos-sdk/blob/master/docs/architecture/adr-023-protobuf-naming.md. From what I understand, if buf is shouting, then something's wrong and the pkg version needs to be bumped.

Reply:
We need to require proto-breaking CI checks...